model/src/w3wavemd.F90

```diff
@@ -1493,6 +1499,8 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
 
             CALL INIT_GET_ISEA(ISEA, JSEA)
 
+            IF ((IOBP_LOC(JSEA).eq.1..or.IOBP_LOC(JSEA).eq. 3).and.IOBDP_LOC(JSEA).eq.1.and.IOBPA_LOC(JSEA).eq.0) THEN
```

Comment:
This if block is not properly indented.

Reply:
Thank you @DeniseWorthen  - Fixing in develop and then will merge in here..